src/main/kotlin/apply/ui/admin/mission/MissionsFormView.kt

```diff
@@ -31,7 +31,11 @@ class MissionsFormView(
     private var recruitmentId: Long = 0L
     private val title: Title = Title()
     private val missionForm: MissionForm by lazy {
-        MissionForm(evaluationService.getAllSelectDataByRecruitmentId(recruitmentId))
+        MissionForm(
+            evaluationService.getAllSelectDataByRecruitmentId(recruitmentId),
+        ) {
+            evaluationService.findEvaluationItems(it)
+        }
```

Comment:
```suggestion
        MissionForm(
            evaluationService.getAllSelectDataByRecruitmentId(recruitmentId)
        ) {
            evaluationService.findEvaluationItems(it)
        }
```

r : 콤마가 불필요하게 남아있네요 👀